conf/routes

```diff
@@ -29,6 +29,11 @@ POST        /admin/trips/edit/:id                              @femr.ui.controll
 GET         /admin/trips/teams                                 @femr.ui.controllers.admin.TripController.teamsGet()
 POST        /admin/trips/teams                                 @femr.ui.controllers.admin.TripController.teamsPost()
 GET         /admin/trips/cities                                @femr.ui.controllers.admin.TripController.citiesGet()
+#add test (Hello World)
+GET         /admin/triagePatients/triagePaitent                 @femr.ui.controllers.admin.TriagePatientController.triagePatient()
+POST        /admin/triagePatients/triagePaitent                 @femr.ui.controllers.admin.TriagePatientController.triagePatient()
```

Comment:
POST is only used when sending a request/data back to the server. I don't think we need this here.